src/execution/exchanges/swap.execute.ts

```diff
@@ -167,7 +167,7 @@ export class SwapExecutionManager {
         error: {
           code: LifiErrorCode.TransactionFailed,
           message: 'Failed while waiting for receiving chain.',
-          htmlMessage: getTransactionFailedMessage(swapProcess),
+          htmlMessage: getTransactionFailedMessage(step, swapProcess.txHash),
```

Comment:
you are passing the `txHash` here but the method expects a `txLink`. is that on purpose?

Reply:
that was a mistake